hedera-node/hedera-app/src/main/java/com/hedera/node/app/tss/handlers/TssVoteHandler.java

```diff
@@ -35,6 +43,9 @@
  */
 @Singleton
 public class TssVoteHandler implements TransactionHandler {
+
+    public static final double CONSENSUS_VOTE_THRESHOLD_ONE_THIRD = 3.0;
```

Comment:
does this assume the total weight of the network is evenly divisible by 3 or what is the significance of setting it to 3.0 ?

Reply:
It doesn't assume that but there is a difference in the threshold comparison if we go with a double here or a long. The proposal didn't seem to indicate using one or the other.